electrotrans-pojo-bl/src/main/java/decavun2/personnel/Person.java

```diff
@@ -53,6 +82,27 @@ public class Person extends ActivatableAbstractEntity<DynamicEntityKey> {
     @Title(value = "Email", desc = "Uniquely identifies a person.")
     @BeforeChange({ @Handler(MaxLengthValidator.class), @Handler(EmailValidator.class) })
     private String email;
+    
+    @IsProperty
+    @MapTo
+    @Required
+    @Title(value = "First Name", desc = "Person's first name.")
+    @BeforeChange(@Handler(NoSpacesValidator.class))
```

Comment:
@Sukhorskyy Do you think it would be sensible to add additional integrity constants for properties `name` and `surname` that would account for consistent letter case? For example, so that user won't be able to enter `MiKe` or `VoLOdymYR` instead of `Mike` and `Volodymyr`. Do you think implementing it as a `@BeforeChange` validator or simply automatically change the input to correct format by using `@AfterChange` or `PersonDao`?

Reply:
I would leave the possibility to enter a capital letter inside the name or surname. There may be situations when a person's name consists of two parts. For instance, _Maria-Kateryna_.